### ISSUE TITLE
Create video tutorial of how to contribute using just the github web interface

### ISSUE THREAD

**Issue by IstoraMandiri** [ethereumclassic/ethereumclassic.github.io#303]
There are contributors who aren't necessarily programmers who shouldn't need to know how to clone repos using git and building the project in development mode.

With the new branching system, it's significantly easier understand how things are actually published.

With this in mind, it makes sense to create a short tutorial explaining how to make contributions without having to download, clone or build anything.

Some examples that might be included

- Adding a blog article
- Updating a language file

Ref #218 #245

**Comment by TheEnthusiasticAs** [comment]
A good idea. I would rise it in discord too. Maybe someone can contribute too - under ethereumclassicwebsite

**Comment by bobsummerwill** [comment]
Right.  Donald and Chris Seberino spring to mind as prolific authors whose blog posts are valuable but for whom GitHub is a little much.  Kevin too.

**Comment by hitchcott** [comment]
Could also probably do a demo of https://gitpod.io/#https://github.com/ethereumclassic/ethereumclassic.github.io

This includes a full IDE and git client, lets you run gatsby in developer mode, opens up ports, so you can do live reloading content.

**Comment by hitchcott** [comment]
https://github.com/ethereumclassic/ethereumclassic.github.io/blob/babel/docs/translate.md
